packages/public/server/src/module/Api/src/ApiManager.php

```diff
@@ -339,6 +341,16 @@ public function getUuidData(UuidInterface $uuid)
         return $data;
     }
 
+    private function getRevisionIds($uuid)
+    {
+        return $uuid
+            ->getRevisions()
+            ->map(function ($rev) {
+                return $rev->getId();
+            })
```

Comment:
I'd prefer
```suggestion
            ->map(function ($revision) {
                return $revision->getId();
            })
```
I try to avoid as many abbreviations as possible as long as it gets not too verbose.